plugins/geo_json_renderer.py

```diff
@@ -23,12 +24,23 @@ async def render_geo_json(
     feature_list = list()
 
     for row in result:
-        # https://geojson.org/geojson-spec.html#id2
-        point = geojson.Point((row['longitude'], row['latitude']))
-        # create a geojson feature
-        feature = geojson.Feature(geometry=point, properties=dict(row))
-        # append the current feature to the list of all features
-        feature_list.append(feature)
+        # Is this a blatant Point object?
+        if 'longitude' in row.keys() and 'latitude' in row.keys():
+            # https://geojson.org/geojson-spec.html#id2
+            point = geojson.Point((row['longitude'], row['latitude']))
+            # create a geojson feature
+            feature = geojson.Feature(geometry=point, properties=dict(row))
+            # append the current feature to the list of all features
+            feature_list.append(feature)
+        
+        # Otherwise, does this have a "the_geom" object, which was used in the old Carto database, which encodes geographical data as a string in the "well-known binary" format?
+        elif 'the_geom' in row.keys():
```

Comment:
Not blocking: This assumes that a result row has either a Point Feature (lat/lon coords) OR Geometry Feature (the_geom). I assume we don't want rows that have both types in our use case? 

FWIW we could easily encode both features for a single row in the `feature_list` result set.

Reply:
Actually, a single `camera` _row_ does have both types - it has lat/long coordinates AND a Geometry feature (the_geom). See: http://localhost:8001/darien.json?_shape=objects&sql=SELECT%20*%20FROM%20cameras

This is because any raw data uploaded to Carto automagically gets a `the_geom` field. (For a camera row, its `the_geom` field basically translates to the same thing as its explicit `latitude` and `longitude` fields.)

Side note: I can actually remove the first `if` case here and just use the `the_geom` to generate Point features, but I _favour checking for explicit data_ (i.e. `latitude` and `longitude` fields) because moving forward we probably won't be auto-generating `the_geom` fields like Carto did.